src/lib.rs

```diff
@@ -110,34 +138,39 @@ impl Build {
             fs::remove_dir_all(&build_dir_path)?;
         }
 
-        log::trace!("Build: Creating new build directory");
-        fs::create_dir(&build_dir_path).context("cannot create build directory")?;
+        log::trace!("Creating new build directory");
+        fs::create_dir_all(&build_dir_path).context("cannot create build directory")?;
 
-        log::trace!("Build: Writing files into build directory");
+        log::trace!("Writing files into build directory");
         fs::write(wasm_js_path, wasm_js).with_context(|| "cannot write js file")?;
         fs::write(wasm_bin_path, wasm_bin).with_context(|| "cannot write WASM file")?;
 
         let mut copy_options = fs_extra::dir::CopyOptions::new();
         copy_options.overwrite = true;
         copy_options.content_only = true;
 
-        log::trace!("Build: Copying static directory into build directory");
-        fs_extra::dir::copy(static_dir_path, build_dir_path, &copy_options)
-            .context("cannot copy static directory")?;
+        if let Some(static_dir) = self.static_dir_path {
+            log::trace!("Copying static directory into build directory");
+            fs_extra::dir::copy(static_dir, build_dir_path, &copy_options)
+                .context("cannot copy static directory")?;
+        }
 
-        Ok(())
+        log::info!("Builded successfully at {}", build_dir_path.display());
+
+        Ok(build_dir_path.to_path_buf())
```

Comment:
better keep the camino type here. As I said, you can convert freely a camino path to a std path but the invert is not true